lib/reevoo-ping.js

```diff
@@ -2,13 +2,22 @@
 //
 // Main event recording class.
 
+import { COLLECTOR_URI } from './config';
+import snowplow from './snowplow';
+import defaultSnowplowConfig from './snowplow/default-config';
 import validate from './validate';
 import Experiences from './events/experiences';
 import Page from './events/page';
 import Badge from './events/badge';
 
 class Client {
-  constructor(opts) {
+  constructor(appId, opts = {}, trackerConfig = {}) {
```

Comment:
This changes the signature of the object constructor. Is not going to break everything that we already have live? I am asking because the first param was previously `opts` instead of an `appId`.